docs/tutorial.rst

```diff
@@ -385,6 +351,57 @@ There may be times when you need to import an existing scratch org that wasn't c
 
 You can hop into a browser logged into any org in your keychain with `cci org browser <org_name>`.
 
+Creating a Connected App
+------------------------
+
+In order to connect persistent orgs such as a Developer Edition, Enterprise Edition, or Sandbox org to CumulusCI, you need to have a Connected App configured in a persistent Salesforce org.
+
```

Comment:
Add a sentence like "You have a choice of whether to create the Connected App from the command line or in Salesforce Setup." otherwise most readers will probably go ahead with the next section without realizing there are 2 options.